ocs_ci/ocs/hsbench.py

```diff
@@ -67,7 +67,7 @@ def create_resource_hsbench(self):
             deploy_pod_status=constants.STATUS_COMPLETED,
         )
 
-    def install_hsbench(self, timeout=2400):
+    def install_hsbench(self, timeout=4200):
```

Comment:
Timeout value is more than 1hr, does the install alone taking this much time? is it because if any slow download of image it's we need more time?

Reply:
I have seen case that some VM clusters (DC-15) take more than 40mins to get the image, therefore increasing the time.  In a normal case, it won't take more than 30 mins.

Reply:
We have to enhance the script for time optimization, do plan in next PRs